src/oc-id/app/controllers/password_resets_controller.rb

```diff
@@ -70,8 +70,11 @@ def update
   private
 
   def user_not_found
-    flash.now[:alert] = "User '#{params[:username]}' not found"
-    @status = :not_found
+    flash.now[:notice] = password_reset_msg
+  end
+
+  def password_reset_msg
+    "If the username you entered exists, you should receive an email shortly"
```

Comment:
I'll confess that i18n didn't occur to me, given that none of the other messages in the controller used it.  I can open a new PR to convert them all over.